mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulSettingsScreen.kt

```diff
@@ -1,21 +1,37 @@
 package ch.epfl.sdp.mobile.state
 
-import androidx.compose.runtime.Composable
-import androidx.compose.runtime.remember
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.runtime.*
 import androidx.compose.ui.Modifier
 import ch.epfl.sdp.mobile.application.authentication.AuthenticatedUser
+import ch.epfl.sdp.mobile.application.chess.ChessFacade
 import ch.epfl.sdp.mobile.ui.setting.SettingScreenState
 import ch.epfl.sdp.mobile.ui.setting.SettingsScreen
 import ch.epfl.sdp.mobile.ui.social.ChessMatch
 import ch.epfl.sdp.mobile.ui.social.Person
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.launch
 
 class AuthenticatedUserProfileScreenState(
```

Comment:
Might Require some Documentation

Reply:
Fixed in 9337db8ae78de9150eed15e056e78fb9165dd16a